dask_ml/model_selection/_incremental.py

```diff
@@ -653,10 +647,10 @@ class IncrementalSearchCV(BaseIncrementalSearchCV):
 
         Alternatively, you can set this to ``"grid"`` to do a full grid search.
 
-    decay_rate : float, default 1.0
+    decay_rate : float, default 0.0
         How quickly to decrease the number partial future fit calls.
         Higher `decay_rate` will result in lower training times, at the cost
-        of worse models.
+        of worse models. If changed, we recommend ``decay_rate == 1``.
```

Comment:
Why exactly 1?

Reply:
It'd be good to note that this is useful for compute-bound (& memory-bound) problems.